.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* nickatsegment eculver
```

Comment:
Nobody else really seemed to perk up on the idea, but LMK if you're keen and I'll add ya. I'm not sure if we can add org teams, or how that works exactly.

Reply:
Yep, agreed. We just need to agree internally on what that team is :)